src/shared_storage.rs

```diff
@@ -116,44 +131,44 @@ impl SharedStoragePoolManager {
         Ok(())
     }
 
-    #[allow(dead_code)]
-    async fn get_account_storage(&self) -> anyhow::Result<Option<StorageView>> {
-        self.rpc_client
-            .view(
-                &self.pool_contract_id,
-                "get_account_storage",
-                serde_json::json!({
-                    "account_id": self.pool_owner_id,
-                }),
-            )
-            .await
-            .map_err(Into::into)
-    }
+    // #[allow(dead_code)]
```

Comment:
Is there a reason why we are allowing dead code here?

Reply:
It's the function that was previously used to get account storage. Commented it out in case the near-fetch version  needed to be reverted. I'll remove it.